app/_locales/en/messages.json

```diff
@@ -5210,6 +5210,12 @@
   "stake": {
     "message": "Stake"
   },
+  "stateCorruptionDetectedNoBackup": {
```

Comment:
we should also copy them in en_GB, right?

Reply:
Done in d873b38c35